packager/src/main/java/io/cdap/hub/GoogleCloudStorageClient.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright © 2016 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+
+package io.cdap.hub;
+
+import com.google.cloud.storage.Storage;
+import com.google.cloud.storage.StorageOptions;
+
+/**
+ * Component to create Google Cloud Storage connection. Can be used for mocking behaviour in tests.
```

Comment:
for mocking behavior in tests i would also glance at how google does it for their GCS plugins in https://github.com/data-integrations/google-cloud

Reply:
https://github.com/data-integrations/google-cloud/blob/develop/src/test/java/io/cdap/plugin/gcp/gcs/StorageClientTest.java 
Looked at this test class, there's nothing about mocking the actual Storage. Nothing either in the other plugins about mocking out the storage.

There's this one apparently that we can use : https://github.com/googleapis/java-storage-nio/blob/master/google-cloud-nio/src/main/java/com/google/cloud/storage/contrib/nio/testing/LocalStorageHelper.java
Will try that out otherwise will just mock the required behaviour with

Reply:
sounds good